src/converters/piral-blazor/README.md

```diff
@@ -131,11 +131,15 @@ export function setup(piral: PiletApi) {
 }
 ```
 
+In this case, you'll also have to install the `blazor` package. `piral-blazor` will use this under the hood to access the Blazor libraries.
+
+It should be noted that to maximize compatibility, the version of the `blazor` package should be identical to the version of Blazor you want to use. (e.g. use `blazor@3.2.1` for Blazor v3.2.1).
```

Comment:
We are only compatible on the minor part, i.e., 3.2.x is compatible with 3.2.x - if we have a bug / issue in our deployment we can then still fix this with the patch - level. The patch level will also always have the latest patch level of Blazor in there (e.g., NPM "blazor" 3.2.10 may be able to retrieve .NET Blazor 3.2.1 if it was the latest 3.2.x release when the NPM package was released).

Reply:
is this a better way of describing it?

> It should be noted that to maximize compatibility, the major and minor version of the `blazor` package should be identical to the major and minor version of .NET Blazor you want to use (e.g. `blazor@3.2.5` would give you the latest .NET Blazor `v3.2.x` release that was available when `blazor@3.2.5` was published). 

I'm struggling a bit with the wordings to make it not too confusing

Reply:
Yeah maybe just note that major and minor versions are always compatible, i.e., `3.2.x` of "blazor" will resolve to .NET's Blazor `3.2` release train. The patch level is not aligned. If a specific patch level is wanted the documentation of the `blazor` package should be consulted (in there we should list the specific pinning).